packages/truffle-box/box.js

```diff
@@ -7,13 +7,7 @@ const fse = require("fs-extra");
 const inquirer = require("inquirer");
 
 function parseSandboxOptions(options) {
-  if (typeof options === "function") {
-    return {
-      name: "default",
-      unsafeCleanup: false,
-      setGracefulCleanup: false
-    };
-  } else if (typeof options === "string") {
+  if (typeof options === "string") {
```

Comment:
So are there places in the code where `typeof options === "function"`? That would mean that `sandbox` would be called with the callback being the first argument I guess? We should make sure that doesn't happen if we remove the check.

Reply:
I guess it looks like you got them all :)